CharacterCreation/03_Feats.txt

```diff
@@ -689,113 +696,182 @@ now working with. This feat can only be taken once per weapon.
-30:	Gain feat "Beware the Wrath of a Patient Man": shots fired as a held reaction have a -1 miss chance.			
-40:	Long Rifle Master. +10% Base damage; Gain feat "Practiced Precision": DC 75% dexterity or sniper skill
-		check to negate the action for changing your aim from one target to another in front of you 
-		(1 successful negation per turn).			
+	These feats represent dedicated training with a particular class of weapon, 
+and offer bonuses based on the relevant Weapons skill (Weapon skills are listed 
+in "06_Skills.txt"). Each feat offers a set of bonuses related to using that type of 
+weapon (and only that type of weapon), and each bonus requires the character to have 
+a number of points in the relevant Weapon skill. A character with 0 points in 
+Weapons - Pistols would gain only one benefit, while 60 points would benefit from 
+them all. The feats are formatted as a list of "X: Benefit", where X is the number of 
+points required in the corresponding Weapon skill to gain the benefit.
 	
-== Proficient: Heavy Weapons ==
-	Skill @
-
-10:	Heavy Weapon Proficient. +5m to range R2 & R3	
-15:	-10% reload miss chance			
-20:	+1m radius to your Suppression Area of Effect			
-25:	Heavy Weapon Expert. -0.5 miss chance @ all ranges			
-30:	Heavy Weapons no longer jam			
-40:	Heavy Weapon Master. +10% Base damage; Gain feat "Effective Fire": when a
-		target is suppressed by your heavy weapon, they must make a DC 62% 
-		Shock save. On failure, they lose two actions at the start of their 
-		next turn.			
-
-== Proficient: Explosives ==
-	Skill @	
-
-10:	Explosives Proficient. +10% explosives use			
-15:	-10% reload miss chance (grenade launchers, rocket launchers, etc.)			
-20:	+5m to all range brackets for explosive launchers (grenade launchers, etc.)			
-25:	Explosives Expert. -0.5 miss chance @ all ranges (grenades, rocket launchers, etc.)			
-30:	+10% to charisma rolls for intimidation when holding any explosive/launcher			
-40:	Explosives Master. +15% Base damage, Gain feat "Get Down!": Allies in blast radius 
-		may make a DC 60 Reflex save to take half damage (before armor) from friendly fire.			
-		
+	Weapon Expertise feats require proficiency with the weapon type, but otherwise 
+have no prerequisites. Note that as all characters are proficient with Unarmed Strike, 
+Melee Expertise effectively has no prerequisite.
+
+== Pistol Expertise ==	
+
+0:	+5m to ranges R1 & R2. If this would cause R2 to be greater than R3, 
+	also set R3 equal to the new R2.
+12:	May use Weapons - Pistol (Int) in place of Stealth for checks made to prevent 
+	enemies from noticing your Pistol fire.
+24:	Reduce the Reload check penalty imposed by the Dual-Wield / Akimbo feats 
+	by 10 when dual-wielding pistols.
+36:	+1 to Accuracy rolls at all ranges.
+48:	+10m to range R3.
+60:	+10% Base damage. If you succeed on an Accuracy roll by at least 3 or 
+	extra-crit, treat the enemy's armor as having APL 0 for that attack.

```

Comment:
How are we wording "extra-crit"?

Reply:
It's a little unfair to ding him for that; it changed in the middle of Turtle doing this work. It's "Extra-Critical" now https://github.com/bleehu/Compound_X/blob/dev/01_02_Combat_Rules.txt#L369

Reply:
But still a valid call